test/Microsoft.AspNetCore.SignalR.Client.Tests/HubConnectionProtocolTests.cs

```diff
@@ -2,12 +2,16 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.Reactive.Linq;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.SignalR.Internal.Protocol;
 using Microsoft.AspNetCore.SignalR.Tests.Common;
 using Microsoft.Extensions.Logging;
 using Newtonsoft.Json;
 using Xunit;
+using System.Reactive;
```

Comment:
Weird... that one was definitely brought in with Ctrl-. ... at least I'm pretty sure it was.